apps/firestorm_web/lib/firestorm_web/earmark/auto_linker.ex

```diff
@@ -4,8 +4,8 @@ defmodule FirestormWeb.Earmark.AutoLinker do
   wrapped in an `a` tag with the link as both content and href..
   """
 
-  # A RegEx to match any URL that has spaces or newlinex on either side of it.
-  @url_regex ~r{(http|ftp|https)://([\w_-]+(?:(?:\.[\w_-]+)+))([\w.,@?^=%&:/~+#-]*[\w@?^=%&/~+#-])?}
+  # A RegEx to match any URL that has spaces or newlines on either side of it.
+  @url_regex ~r{([ \n]+|^)(http|ftp|https)://([\w_-]+(?:(?:\.[\w_-]+)+))([\w.,@?^=%&:/~+#-]*[\w@?^=%&/~+#-])?}
```

Comment:
This is a way of getting around the problem because the markdown link generation doesn't leave a space when it generates the HTML?